.github/workflows/build.yml

```diff
@@ -68,37 +67,51 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - name: install system dependencies
-      run: sudo apt-get update && sudo apt-get install libelf-dev mpich libmpich-dev libomp-9-dev libsystemd-dev
-    - name: install python dependencies
-      uses: py-actions/py-dependency-install@v2
-      with:
-        path: scripts/requirements.txt
+      run: sudo apt-get update && sudo apt-get install libelf-dev mpich libmpich-dev libomp-9-dev libsystemd-dev libgirepository1.0-dev
+    - name: install base and service dir python dependencies
+      run: |
+           python3 -m pip install --upgrade pip setuptools wheel pep517
```

Comment:
I guess we could go back to the standard github script now that the requirements.txt no longer reference geopmdpy?

Reply:
No, we can't. The PyGobject requirement is problematic, and requires pip517 in order for pip to be able to successfully complete the source build.  There's no option with the standard github script thing to use pep517.